x-pack/plugins/console_extensions/spec/generated/ilm.delete_lifecycle.json

```diff
@@ -0,0 +1,11 @@
+{
+  "ilm.delete_lifecycle": {
+    "methods": [
+      "DELETE"
+    ],
+    "patterns": [
+      "_ilm/{lifecycle}"
```

Comment:
I would call the placeholder `policy` here (and anywhere else where we ask for a lifecycle policy name) instead of `lifecycle`. Not sure if the placeholder is shown to the user though?

Reply:
@colings86 the placeholder is not shown to the user.  Those files are generated from the Elasticsearch REST specs, and that is where the placeholder is coming from?  Can someone fix the issue in ES and I'll regenerate?  I don't want to hand edit these ever, since they live in the "generated" directory.

Reply:
@bmcconaghy I opened https://github.com/elastic/elasticsearch/pull/33507 in the ES repo for the ask above